lib/defaults/defaults.go

```diff
@@ -666,15 +666,33 @@ const (
 	// MaxRouterIdleConnsPerHost defines tha maximum number of idle connections for "opsroute" transport
 	MaxRouterIdleConnsPerHost = 5
 
-	// KubernetesHostnameLabel is the name of kubernetes label what contains host's IP
-	KubernetesHostnameLabel = "kubernetes.io/hostname"
-
 	// KubernetesRoleLabel is the Kubernetes node label with system role
 	KubernetesRoleLabel = "gravitational.io/k8s-role"
 
 	// KubernetesAdvertiseIPLabel is the kubernetes node label of the advertise IP address
 	KubernetesAdvertiseIPLabel = "gravitational.io/advertise-ip"
 
+	// KubernetesNodeRoleLabelFormat is the kubernetes node label shown in `kubectl get nodes`
+	KubernetesNodeRoleLabelFormat = "node-role.kubernetes.io/%v"
+
+	//
+	// Kubernetes Well Known Labels
+	// https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/
+	//
+
+	// KubernetesArchLabel is the kubernetes well-known label for arch
+	KubernetesArchLabel = "kubernetes.io/arch"
```

Comment:
nit: I would reference the label consts in api:
```go
import "k8s.io/api/core/v1"
...

KubernetesArchLabel = v1.LabelArchStable
// ditto for other labels
```

Reply:
Hmm, in this case, I think I would go the direction of just using the kubernetes constants directly, not sure we buy much by aliasing them into defaults.